Resources/doc/5-encoder-service.md

```diff
@@ -4,7 +4,7 @@ JWT encoder service customization
 This bundle comes with a [`namshi/jose`](https://github.com/namshi/jose) library based token encoder as it uses SSH keys to encrypt and decrypt data.
 If this doesn't suit your needs, you can replace it with your own encoder service. Here's an example implementing a [`nixilla/php-jwt`](https://github.com/nixilla/php-jwt) library based encoder.
 
-#### Create the encoder class
+### Create the encoder class
 
 ``` php
 // src/Acme/Bundle/ApiBundle/Encoder/NixillaJWTEncoder.php
```

Comment:
+ add namespace to controllers as `AppBundle\Controller` for all code examples 👍